canonical-web-and-design/maas-ui
# feat(ui): update new-release notifications

## Done

- Replace the opt-out menu with a settings link on new release notifications.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Create a new release notification (instructions are here: https://github.com/canonical-web-and-design/MAAS-squad/issues/2065#issue-669660220).
- Check that the notification has a "See settings" link instead of the opt-out menu.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1507
